websites/C/Coub/presence.ts

```diff
@@ -312,26 +301,28 @@ const pages: PageContext[] = [
 	],
 	presence = new Presence({
 		clientId: "818598086984728576",
-	});
-
+	}),
+	presenceImageKeys = {
+		PLAY: "playx1024",
+		PAUSE: "pausex1024",
+	};
+function getStrings(newLang?: string) {
+	return presence.getStrings(
+		{
+			browsing: "presence.activity.browsing",
+			watching: "presence.playback.playing",
+			watchVideo: "general.buttonWatchVideo",
+			viewProfile: "general.buttonViewProfile",
+		},
+		newLang
+	);
+}
 let currentLang: string, localizedStrings: { [key: string]: string };
```

Comment:
```suggestion
let currentLang: string, localizedStrings: Awaited<typeof getStrings>;
```

Reply:
doesnt work, getStrings is a function.
defined the strings in a const and ref the type to that

Reply:
Yes it does work, I just forgot something. Should be like this:
`Awaited<ReturnType<typeof getStrings>>`
Please change this